sos/report/plugins/ceph_mgr.py

```diff
@@ -36,33 +36,12 @@ class CephMGR(Plugin, RedHatPlugin, UbuntuPlugin):
 
     plugin_name = 'ceph_mgr'
     profiles = ('storage', 'virt', 'container', 'ceph')
-    files = ('/var/lib/ceph/mgr/*', '/var/lib/ceph/*/mgr*')
+    files = ('/var/lib/ceph/mgr/*', '/var/lib/ceph/*/mgr*',
+             '/var/snap/microceph/common/data/mgr/*')
     containers = ('ceph-(.*-)?mgr.*',)
 
     def setup(self):
-
-        self.add_file_tags({
-            '/var/log/ceph/(.*/)?ceph-mgr.*.log': 'ceph_mgr_log',
-        })
-
-        self.add_forbidden_path([
-            "/etc/ceph/*keyring*",
-            "/var/lib/ceph/**/*keyring*",
-            "/var/lib/ceph/**/osd*",
-            "/var/lib/ceph/**/mon*",
-            # Excludes temporary ceph-osd mount location like
-            # /var/lib/ceph/tmp/mnt.XXXX from sos collection.
-            "/var/lib/ceph/**/tmp/*mnt*",
-            "/etc/ceph/*bindpass*",
-        ])
-
-        self.add_copy_spec([
-            "/var/log/ceph/**/ceph-mgr*.log",
-            "/var/lib/ceph/**/mgr*",
-            "/var/lib/ceph/**/bootstrap-mgr/",
-            "/run/ceph/**/ceph-mgr*",
-        ])
-
+        microceph_pkg = self.policy.package_manager.pkg_by_name('microceph')
```

Comment:
So far usage of this variable is bit superfluous: you set it once and use it once, now it is easier to have directly `if not self.policy.package_manager.pkg_by_name('microceph'):`.

But assuming this can evolve in time and this code can be seen with more clarity, no complaint.